### PR TITLE
fix: Improve onboarding Availability

### LINKED ISSUE #10667: mobile: onboarding availability needs love
image

also, lets make cards full width on **mobile**, no margin

image

### PR DESCRIPTION
**Pull request by anikdhabal**
## What does this PR do?

Fixes #10667
Before:-
Screenshot 2023-08-10 041923
Screenshot 2023-08-10 041946

Now:-
Screenshot 2023-08-10 041429
Screenshot 2023-08-10 014938

### CODE REVIEW

**Review by Udit-takkar — DISMISSED** [review]
LGTM

File: packages/features/schedules/components/Schedule.tsx [hunk 1]
```diff
@@ -241,10 +240,10 @@ const RemoveTimeButton = ({
   );
 };
 
-const TimeRangeField = ({ className, value, onChange }: { className?: string } & ControllerRenderProps) => {
+const TimeRangeField = ({ value, onChange }: ControllerRenderProps) => {
   // this is a controlled component anyway given it uses LazySelect, so keep it RHF agnostic.
   return (
-    <div className={className}>
```
Comment (Udit-takkar) [inline packages/features/schedules/components/Schedule.tsx:247]: why did you remove className? 
you can do 
<div className={classNames("flex flex-row gap-1",className)}>

Comment (anikdhabal) [reply packages/features/schedules/components/Schedule.tsx:247]: Ahh that's true. I will fix it.Thank you

Comment (anikdhabal) [reply packages/features/schedules/components/Schedule.tsx]: @Udit-takkar Done.

**Review by sean-brydon — APPROVED** [review]

**Pull request merged**